src/compass/defaults/s1_cslc_geo.yaml

```diff
@@ -17,6 +17,8 @@ runconfig:
       dynamic_ancillary_file_group:
           # Digital elevation model
           dem_file:
+          # Description for DEM used for the processing
+          dem_description: DEM description was not provided.
```

Comment:
Have you tested this? Shouldn't this be the following:

```suggestion
          dem_description: 'DEM description was not provided.'
```

Reply:
confirmed that it works without the quotation marks.
<img width="812" alt="Screenshot 2023-08-21 at 19 01 22" src="https://github.com/opera-adt/COMPASS/assets/27862199/dbd30410-22ba-46ae-a84e-9d156d24f47d">